.github/workflows/rust.yml

```
@@ -81,24 +81,45 @@
+          path: localrelayer
+      - name: Setup docker
+        run: |
+          curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+          chmod +x /usr/local/bin/docker-compose
+          docker-compose --version
+      - name: Setup localrelayer
+        run: |
+          cd ./localrelayer
+          make start-chains
+          make start-validators
+      - uses: actions-rs/toolchain@v1
```

actionlint

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)